patchlevel.h

```diff
@@ -39,9 +39,26 @@ Instead use one of the version comparison macros.  See C<L</PERL_VERSION_EQ>>.
 /* do not adjust the whitespace! Configure expects the numbers to be
  * exactly on the third column */
 
-#define PERL_REVISION	5		/* age */
-#define PERL_VERSION	33		/* epoch */
-#define PERL_SUBVERSION	1		/* generation */
+/* 
+   You should avoid using these variables directly
+   and prefer using the compare functions.
+       - PERL_VERSION_EQ
+       - PERL_VERSION_NE
+       - PERL_VERSION_LT
+       - PERL_VERSION_GT
+       - PERL_VERSION_LE
+       - PERL_VERSION_GE
+*/
+#define PERL_VERSION_MAJOR        5
+#define PERL_VERSION_MINOR       33
+#define PERL_VERSION_PATCH        1
+#define PERL_VERSION_RELEASE      0 /* for future usage */
+#define PERL_VERSION_BUILD        0 /* for future usage */
```

Comment:
I'm cautious about defining these new "for future use" ones, without having a clearer plan of what possible future use they might have.